src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -0,0 +1,82 @@
+package org.folio.rest_v2
+
+import org.folio.models.OkapiTenant
+import org.folio.utilities.RequestException
+
+class Eureka extends Authorization {
+
+  Eureka(Object context, String okapiDomain, boolean debug = false) {
+    super(context, okapiDomain, debug)
+  }
+
+  boolean isApplicationRegistered(String applicationId) {
+
+    String url = generateKongUrl("/applications/${applicationId}")
```

Comment:
@DmytrMoroz 

I can't find **generateKongUrl** function definition in your Pull request ;)